docs/installation.md

```diff
@@ -141,11 +141,9 @@ petstore.yaml: validated in 51ms
 Woohoo! Your OpenAPI definition is valid. 🎉
 ```
 
-## Post-installation
+## Set up tab completion
 
-### Set up tab completion
-
-To set up tab completion for the terminal, generate the completion script with the command:
+After a successful install, you should set up tab completion for the terminal. Generate the completion script using the following command:
```

Comment:
question: Do you think we could somehow indicate that this is not required? Tab completion is a convenient option that makes using OpenAPI CLI easier and faster, but it's not like the tool won't work without it. Our current docs are not great at indicating this...